website/src/main/webapp/results.html

```diff
@@ -6,13 +6,16 @@
     <script src="prototype_script.js"></script>
   </head>
   
-  <body id="body" onload="addListener();">
+  <body id="body" onload="setUpPrototype();">
```

Comment:
```suggestion
  <body id="body" onload="addListener(); setUpPrototype();">
```
addListener() adds listener for the enter keyup